scripts/youtube.js

```diff
@@ -446,14 +446,25 @@ H5P.VideoYouTube = (function ($) {
       }
 
       // Use as much space as possible
-      $wrapper.css({
+      var defaultSize = {
         width: '100%',
         height: '100%'
-      });
+      };
+
+      $wrapper.css(defaultSize);
+      $('iframe', $wrapper).css(defaultSize);
 
       var width = $wrapper[0].clientWidth;
       var height = options.fit ? $wrapper[0].clientHeight : (width * (9/16));
 
+      if (!width) {
+        width = 1;
+
+        setTimeout(function() {
+          self.trigger('resize');
+        }, 500);
```

Comment:
This will cause multiple endless loops of resizing until the browser eventually crashes.

Reply:
What do you mean? 
Settimeout() doesn't do any loops. This method calls a function after a specified number of milliseconds.

Reply:
When you trigger the resize event inside the resize handler you'll get an endless loop, sure it will only be looped every 500ms but it's still a code smell.